dev/deps/spark-deps-hadoop-2.7

```diff
@@ -6,6 +6,7 @@ ST4-4.0.4.jar
 activation-1.1.1.jar
 aircompressor-0.10.jar
 algebra_2.12-2.0.0-M2.jar
+amqp-client-5.5.3.jar
```

Comment:
Do we need this? If not, can we exclude this?

Reply:
This seems to be a new compile dependency from `metrics-graphite`.

Reply:
It does not like we need it, as it is for a "rabbit-mq client to a Carbon server". BTW, the code was there also in dropwizard 3.1.5, but somehow the dependency did not come up.